### DISCUSSION TITLE
Define sets of workspaces for each monitor

### DISCUSSION THREAD

**Discussion by arminveres** [glzr-io/glazewm#228] [Q&A]
Is it already possible do define a set of workspaces for each monitor?
As it stands right now, I have 3 monitors, resulting in 3 workspaces already being taken out of the 9/10.
I would like to avoid having a workspace 20, being only accessible by some other key combination than the simple 'modkey + number'

**Comment by HolbyFPV** [comment]
This is an interesting idea.  I also run 3 monitors and sympathize with your use case.  

I'm not entirely sure what your proposing though.  Would each monitor get it's own 1-9 workspaces?  If that's the case, how would alt+1 know which monitor to open the 1st workspace on?  Maybe if you provide some more examples of how this would work, or even an example of how you might configure this in the yaml, then I might get my head wrapped around this.

**Reply by arminveres** [reply]
On my home workstation I use AwesomeWM. It uses the currently focused monitor, to know, which 'workspace 1' is meant. You then can switch to another monitor whereas the prefixes only apply there.
I haven't looked too much at this code base, but since it is possible to send workspaces to other monitors, it should be possible in theory.

**Reply by HolbyFPV** [reply]
Good to know.  Down the line I was intending on investigating AwesomeWM for their auto layout functionality.  Whenever I do that I'll have to check out exactly how they are handling this.  Thanks for response.
